### PR TITLE
Added to opsgenie module region selection

### PR DESCRIPTION
**Pull request by l13t**
According to OpsGenie API documentation:

`If using the EU instance of Opsgenie, the URL needs to be https://api.eu.opsgenie.com for requests to be successful.`

P.S.
I'm not developer but I tried to do all my best to do this change :)

### CODE REVIEW

File: modules/opsgenie/settings.go [hunk 1]
```diff
@@ -13,6 +13,7 @@ type Settings struct {
 	common *cfg.Common
 
 	apiKey                 string   `help:"Your OpsGenie API token."`
+	isEurope               bool     `help:"Defines if European servers should be used." optional:"true"`
```
Comment (Seanstoppable) [inline modules/opsgenie/settings.go]: We could also consider an optional api url, which some other modules have. This means that if OpsGenie sets up another one, this module will just immediately support it.

Comment (senorprogrammer) [reply modules/opsgenie/settings.go]: Ideally people wouldn't have to know the API URN to use, that's a point of friction that we can abstract away (I believe the other cases where they do, those are for custom installations of services).

I like this approach for now, with one suggestion: changing it to `useEuropeanServiceRegion`. This fits with the naming that OpsGenie uses in their documentation.

Thoughts?

Comment (Seanstoppable) [reply modules/opsgenie/settings.go]: Fair point. If we want to abstract, I would consider abstracting to a region, with the region defaulting to US over a boolean. Better backwards compatibility if they expand and, if not, sufficiently understandable verbosity.

Comment (senorprogrammer) [reply modules/opsgenie/settings.go]: I like the sound of that. Extensible without the potential proliferation of booleans.
```
    opsgenie:
      apiKey: "abc-123"
      region: ["america" | "europe"]
      ...
```
@l13t you say you're not a developer. You've done a great job so far. Are you comfortable implementing this addition, or would you prefer one of us tackle it?

Comment (l13t) [reply modules/opsgenie/settings.go]: @senorprogrammer, sure. I'll rewrite the code as you suggested. I thought about such a way but was not sure about how to limit the number of choices in the variable (to have only `us` and `eu` for now.).

**Comment by l13t** [comment]
@senorprogrammer new version pushed.

**Comment by Seanstoppable** [comment]
This looks great!

**Comment by senorprogrammer** [comment]
This will be merged in today.

**Pull request merged**